pkg/util/etcdutil/member.go

```diff
@@ -126,3 +127,19 @@ func (ms MemberSet) ClientURLs() []string {
 	}
 	return endpoints
 }
+
+func GetIDFromName(name string) (int, error) {
```

Comment:
I think we should figure out it now. this is more confusing now since we pull ID into etcd pkg.